backend/alembic/versions/89f2d9a6ca5e_add_users_table.py

```
@@ -0,0 +1,53 @@
+down_revision: Union[str, Sequence[str], None] = '84d99442d033'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('password', sqlmodel.sql.sqltypes.AutoString(), nullable=True))
+    op.add_column('users', sa.Column('google_id', sqlmodel.sql.sqltypes.AutoString(), nullable=True))
+    op.alter_column('users', 'created_at',
+               existing_type=postgresql.TIMESTAMP(),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)